.travis.yml

```diff
@@ -11,9 +11,14 @@ otp_release:
 install:
         - make
         - make test-compile
+        - travis_retry pip install --user codecov
 script:
         - make test
         - make dialyzer
 
 after_success:
         - make coverage-report
+        - codecov
+
+env:
+          CODECOV_TOKEN="3b9c25a9-e523-4509-8add-023a3acd9c76"
```

Comment:
I don't think adding such tokens to version control (in plain text) is a good idea. Usually such data are set as encrypted env vars in travis-ci config.